.github/workflows/ci.yml

```diff
@@ -65,7 +69,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        platform: [windows-latest, macos-latest, ubuntu-20.04]
+        platform: [windows-latest, macos-13, ubuntu-20.04]
```

Comment:
As we have only observed the failures for `fmt-clippy` so far, I think we should just change it there. The tests still passed on `macos-latest` Without issue. So I think we should still maintain the coverage of aarch64, and I don't think any recommendations clippy can provide diverge for aarch64 in any way that would matter to the quality of the codebase.

Reply:
Great idea, that makes sense!